gazebo_ros/test/ros_network/ros_api_checker

```diff
@@ -188,7 +190,7 @@ if __name__ == '__main__':
     # amount of time when we're downloading models.
     while rospy.Time.now().to_sec() == 0.0:
         print('Waiting for Gazebo to start...')
-        time.sleep(1.0)
+        time.sleep(0.1)
```

Comment:
nit, I would avoid changing this timeout unless shortening is necessary for Noetic/Python 3

Reply:
Restored sleep in 4b50082